app/views/reports/regions/index.html.erb

```diff
@@ -13,7 +13,7 @@
     <div class="col-sm-5 col-lg-7">
       <% @organizations.each do |organization| %>
       <section>
-        <div class="card">
+        <div class="card org-card">
```

Comment:
@rsanheim what does the `org-card` class do?

Reply:
It lets us better target these classes in some of our integration tests - see https://github.com/simpledotorg/simple-server/pull/1378/files#diff-6f0d4ecd603cef429123ac37759e41fe.